src/events/templates/events/series_pass.html

```
@@ -0,0 +1,317 @@
+<!DOCTYPE html>
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)